snyk/cli
# fix: prevent --maven-aggregate-project & --project-name combo

#### What does this PR do?
prevents the use of `--maven-aggregate-project`  with  `--project-name`. This combination isn't viable because `--maven-aggregate-project` produces multiple projects. Incudes test.

#### How should this be manually tested?
build and run `snyk test --maven-aggregate-project --project-name=blah`
You should see the unsupported arg combo message.
